### ISSUE TITLE
Improved error message for passed in events

### ISSUE THREAD

**Issue by yuichiu416** [fullcalendar/fullcalendar#6247]
### Checklist

Please mark these items with an [x]

  - [x] I've already searched through [existing tickets](https://fullcalendar.io/issues)
  - [x] Other people will find this feature useful

### Feature Description

When we pass events or  initialEvents prop
Maybe not all values are valid for each event.
Ex. invalid timestamp, such as '2021-04-14 06:50'
This is a valid string for JS Date but not for the Fullcalendar library.

It will fail silently without any error messages.
Will be helpful if it can show invalid timestamp or something

I was upgrading our legacy code and obviously the format was valid a long time ago but not anymore.

Thanks!

**Comment by ND22Dave** [comment]
I agree, having something like a debug mode that writes messages to the console would be helpful!
